### PR TITLE
Update for using CoreML

### PR DESCRIPTION
**Pull request by Skyline-23**
- Update for Using CoreML (Model auto generated by Xcode)
- Add simple tokenizer
- Make Predict function

### CODE REVIEW

File: Sources/swift-zenz-coreml/swift_zenz_coreml.swift [hunk 1]
```diff
@@ -3,68 +3,117 @@
...
-            }
-            return nil
-        }
+    // 초기화
+    // Initializer
+    init(vocabFile: String, configFile: String) {
+        // 번들에서 파일 URL 가져오기
+        // Get the file URLs from the bundle
+        let vocabURL = Bundle.main.url(forResource: vocabFile, withExtension: "json")!
+        let configURL = Bundle.main.url(forResource: configFile, withExtension: "json")!
```
Comment (ensan-hcl) [inline Sources/swift-zenz-coreml/swift_zenz_coreml.swift]: Please use `Bundle.module.url` instead of `Bundle.main.url` as it works better in Swift Package

**Review by ensan-hcl — CHANGES_REQUESTED** [review]
@Skyline-23 Thank you for your great PR!

Can I ask you to change README.md as follows?

````diff
## Run

```bash
- swift test
+ xcodebuild -scheme swift-zenz-coreml -destination "platform=macOS,name=Any Mac" -configuration Release test
```
````

File: Sources/swift-zenz-coreml/swift_zenz_coreml.swift (hunk 1 above)
Comment (Skyline-23) [reply Sources/swift-zenz-coreml/swift_zenz_coreml.swift]: It look great! I fixed it [bfcca57](https://github.com/ensan-hcl/swift-zenz-coreml/pull/1/commits/bfcca5712f85632b914d99655e235493a8e34a31)

**Comment by Skyline-23** [comment]
@ensan-hcl I fixed it on this commit [bcca8ba](https://github.com/ensan-hcl/swift-zenz-coreml/pull/1/commits/bcca8bad9ffddc81839a04d121ba146fbf7ec269). Thanks 🙏

**Comment by ensan-hcl** [comment]
Thanks!!

**Pull request merged**